tests/pylint/test_imports.py

```diff
@@ -121,3 +121,40 @@ def test_bad_import(
         ),
     ):
         imports_checker.visit_importfrom(import_node)
+
+
+def test_bad_root_import(linter: UnittestLinter, imports_checker: BaseChecker) -> None:
```

Comment:
```suggestion
def test_root_import(linter: UnittestLinter, imports_checker: BaseChecker) -> None:
```
I would rename it and add a few good examples, too. See next comment.

Reply:
Done - split into two separate test functions